sensorlib/src/sensor/SemanticLidarSensor.py

```diff
@@ -101,7 +101,7 @@ def compute_detected_objects(self):
         timestamp, hitpoints = self.__data_collector.get_carla_lidar_hitpoints()
 
         # Compute data needed for occlusion operation
-        actor_angular_extents = self.compute_actor_angular_extents(detected_objects)
+        #actor_angular_extents = self.compute_actor_angular_extents(detected_objects)
```

Comment:
Are we removing this because we removed orientation? Lets leave a TODO comment here stating that we commented this out because we are assuming velocity vector represents orientation